.github/workflows/test.yml

```diff
@@ -35,9 +35,10 @@ jobs:
         env:
           SENTRY_PYTHON2: ${{ matrix.py2 == '1' || '' }}
         run: |
+          echo "Testing clobber-y upgrade"
```

Comment:
This is the initial install, not an upgrade 😀

Reply:
Addressed in 8f39262d443266d043ae0e61f6b77aa26e2b3f20.